src/guides/v2.4/graphql/mutations/create-requisition-list.md

```diff
@@ -0,0 +1,86 @@
+---
+group: graphql
+title: createRequisitionList mutation
+b2b_only: true
+---
+The `createRequisitionList` mutation creates a requisition list for the logged in customer.
+
+This mutation requires a valid [customer authentication token]({{page.baseurl}}/graphql/mutations/generate-customer-token.html).
+
+{:.bs-callout-info}
+Use the [`storeConfig` query]({{page.baseurl}}/graphql/queries/store-config.html) with the following attributes to determine whether requisition lists are supported:
+
+*  `btob_website_configuration_requisition_list_active`
```

Comment:
Delete this line. Since there was only one attribute I moved it to the previous paragraph. 

Add this attribute to the `src/_includes/graphql/store-config.md` topic.